packages/mui-base/src/unstable_useNumberInput/useNumberInput.ts

```diff
@@ -380,5 +422,6 @@ export function useNumberInput(parameters: UseNumberInputParameters): UseNumberI
     isIncrementDisabled,
     isDecrementDisabled,
     inputValue: dirtyValue,
+    inputRef,
```

Comment:
Same question here that I had here: https://github.com/mui/material-ui/pull/39600#issuecomment-1781084822

Reply:
Reverted the ref for now, we can add it when a specific need arises

(It was only marginally related to this PR and it turns out I didn't really need it)